src/Normalizer/FieldItemNormalizer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\metatag\Normalizer\FieldItemNormalizer.
+ */
+
+namespace Drupal\metatag\Normalizer;
+
+use Drupal\Core\Field\FieldItemInterface;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Drupal\serialization\Normalizer\NormalizerBase;
+
+/**
+ * Converts the Metatag field item object structure to METATAG array structure.
+ */
+class FieldItemNormalizer extends NormalizerBase {
+
+  /**
+   * The interface or class that this Normalizer supports.
+   *
+   * @var string
+   */
+  protected $supportedInterfaceOrClass = 'Drupal\metatag\Plugin\Field\FieldType\MetatagFieldItem';
+
+  /**
+   * Implements \Symfony\Component\Serializer\Normalizer\NormalizerInterface::normalize()
+   */
+  public function normalize($field_item, $format = NULL, array $context = array()) {
+    $values = $field_item->toArray();
+    if (isset($context['langcode'])) {
+      $values['lang'] = $context['langcode'];
+    }
+
+    $tags = $values;
+
+    if ($format == 'json' || $format == 'hal_json') {
+        $tags = array();
```

Comment:
These lines are indented incorrectly, they should be intended by two spaces instead of four.